x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/details.ts

```diff
@@ -268,28 +271,32 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
           }) => date
         );
 
-        log.debug(`API RESULT: ${JSON.stringify(dateOnAllInstances)}`);
+        log.debug(
+          `API RESULT: ${Object.entries(dateOnAllInstancesFromApiResponse)
+            .map(([id, date]: [string, number]) => `${id}: ${moment(date).utc()}`)
```

Comment:
nit: I would have thought you wouldn't need the typing here, given the typing above in `mapValues()`